src/pages/profile-page/profile-page.js

```diff
@@ -90,92 +114,96 @@ const ProfilePage = () => {
     }
   }, [userData, resetForm]);
 
-  const getTextFields = (textFields) =>
-    Object.keys(textFields).map((name) => (
+  const getTextFields = useCallback(
+    (textFields) =>
+      Object.keys(textFields).map((name) => (
+        <TextField
+          key={name}
+          type='text'
+          name={name}
+          variant={TEXT_FIELD_VARIANT.OUTLINED}
+          value={values[name]}
+          InputProps={
+            name === 'phoneNumber'
+              ? {
```

Comment:
use && operator

Reply:
It should have an object prop, otherwise, we will get a warning